src/jio-navbar.ts

```diff
@@ -292,6 +292,12 @@ export class Navbar extends LitElement {
     this.menuToggled = !this.menuToggled;
   }
 
+  private _handleSearchboxClick() {
+    if (this.menuToggled) {
+      this._clickCollapseButton();
```

Comment:
if it is not appropriate to use this._clickCollapseButton(); then I shall replace it with this.menuToggled = false; . If both these approaches are wrong then please do advise as to how I could approach to fix the issue https://github.com/jenkins-infra/jenkins.io/issues/6046